dpdk.spec

```diff
@@ -147,11 +147,12 @@ as L2 and L3 forwarding.
 %endif
 
 %prep
-%autosetup -n dpdk-stable-%{?commit0:%{commit0}}%{!?commit0:%{ver}} -p1
+%autosetup -n dpdk-%{?commit0:%{commit0}}%{!?commit0:%{ver}} -p1
 
 %build
+pip install pyelftools
 %meson --includedir=%{incdir} -Ddrivers_install_subdir=%{pmddir} \
-                              -Ddisable_drivers=compress/isal    \
+                              -Ddisable_drivers=compress/isal
 # sadly this results in an error
 #                              -Denable_docs=true
```

Comment:
Is this still true?  We should try enabling this again and see if they have fixed their issues with it.

Reply:
```
[1154/1377] /usr/bin/python3 ../buildtools/call-sphinx-build.py /usr/bin/sphinx-build 21.02.0 /builddir/build/BUILD/dpdk-21.02/doc/guides /builddir/build/BUILD/dpdk-21.02/x86_64-redhat-linux-gnu/doc/guides
FAILED: doc/guides/html 
/usr/bin/python3 ../buildtools/call-sphinx-build.py /usr/bin/sphinx-build 21.02.0 /builddir/build/BUILD/dpdk-21.02/doc/guides /builddir/build/BUILD/dpdk-21.02/x86_64-redhat-linux-gnu/doc/guides
Traceback (most recent call last):
  File "../buildtools/call-sphinx-build.py", line 22, in <module>
    if StrictVersion(ver) >= StrictVersion('1.7'):
  File "/usr/lib64/python3.6/distutils/version.py", line 40, in __init__
    self.parse(vstring)
  File "/usr/lib64/python3.6/distutils/version.py", line 137, in parse
    raise ValueError("invalid version number '%s'" % vstring)
ValueError: invalid version number 'these.'
```